cli/README.md

```diff
@@ -57,6 +57,95 @@ yo mern:component Sidebar --stateless
 
 Produces `app/components/Sidebar.js`.
 
```

Comment:
Seems fair! I will add a global note on the matter, so it serves for all generators (component, crud and so on).